ChangeLog

```diff
@@ -1,3 +1,47 @@
+Wed, 26 Jul 2017 15:27:20 +0200 IvozProvider Team <dev@list-ivozprovider.irontec.com>
+
+    * IvozProvider 1.5 released
+
+    * Administration Portal
+        - Added register status in Terminals, Friends and Accounts lists
+        - Billable calls now display used Peering Contract
+        - Created companies have external calls limited by default (2 in retail, 10 in vPBX)
+        - Match Lists: New section for grouping specific numbers or numeric patterns
+        - Company networks can now have any mask
+        - Outgoing DDI Rules: New section for dynamic assigning presentation DDI
+            based on call destination.
```

Comment:
Add entry for #234 like:

_- Allow using a uploaded/recorded locution as voicemail message by user._